client/gutenberg/extensions/map/view.js

```diff
@@ -0,0 +1,31 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+import './style.scss';
+import component from './component.js';
+import { settings } from './settings.js';
+import FrontendManagement from 'gutenberg/extensions/shared/frontend-management.js';
+import apiFetch from '@wordpress/api-fetch';
+
+window.addEventListener( 'load', function() {
+	// Do not initialize in editor.
+	if ( window.wp.editor ) {
+		return;
+	}
```

Comment:
@enejb with latest SDK changes, are `view.js` scripts anymore loaded in the editor?

Reply:
Noted for future discussion.